### ISSUE TITLE
Setting size parameter to Scroll Search Helper throws ResponseError

### ISSUE THREAD

**Issue by tioback** [elastic/elasticsearch-js#1282]
## 🐛 Bug Report

When working with the Scroll Search Helper, passing the `size` parameter would result in a `ResponseError: illegal_argument_exception` being thrown.

## To Reproduce

```js
const { Client } = require('@elastic/elasticsearch')
const client = new Client({ node: 'http://localhost:9200' })

const scrollSearch = client.helpers.scrollSearch({
  index: 'stackoverflow',
  size: 500
  body: {
    query: {
      match: {
        title: 'javascript'
      }
    }
  }
})

for await (const result of scrollSearch) {
  console.log(result)
}
```
## Expected behavior

I expected the search to return batches of 500 records, which works fine for the first batch (provided by the inner `client.search()` call).
However, the following batch fails (provided by the `client.scroll()` call).

```js
StackTrace: ResponseError: illegal_argument_exception
    at IncomingMessage.<anonymous> (/dev/workspace/elasticsearch-helper-sandbox/node_modules/@elastic/elasticsearch/lib/Transport.js:231:25)
    at IncomingMessage.emit (events.js:228:7)
    at endReadableNT (_stream_readable.js:1185:12)
    at processTicksAndRejections (internal/process/task_queues.js:81:21)
```

## Your Environment

- *node version*: 12.14.1
- `@elastic/elasticsearch` *version*: >=7.8.0
- *os*: Mac

**Comment by tioback** [comment]
Upon investigation, the problem is that the same `querystring` is being used by the helper to handle both the `search()` and the `scroll()` calls.

I'll open a pull request with a fix proposal.

**Comment by delvedor** [comment]
Closed in #1284.
